auto-setup.sh

```diff
@@ -34,18 +34,37 @@ else
     exit 1
 fi
 
+# Define the undesired path
+old_path="/usr/local/bin/devxp-files"
+new_path="$HOME/.local/devxp-files"
+
+# Check if the current directory is the old directory
```

Comment:
Comment that this can be removed after some grace period. Maybe create a debt ticket as well.